### PR TITLE
Use a standard GitHub .gitignore file

### LINKED ISSUE #7: Building the site makes the Git working tree dirty
After I build the site, my Git working tree is [dirty](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefdirtyadirty). Here are some steps to reproduce:

1. Make sure that your Git working tree is [clean](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefcleanaclean). In other words, when you run `git status`, it should output something like this:

    ```console
    $ git status
    On branch main
    Your branch is up to date with 'upstream/main'.

    nothing to commit, working tree clean
    ```

2. Follow [the README’s instructions for building and previewing the site locally](https://github.com/smartkidsllc/smartkidsllc.github.io/tree/a07ce07b980b40c0720c8bec3e4b83762e59bc00?tab=readme-ov-file#building-and-previewing-the-site-locally).

3. Once the Web page loads in your browser, close out of it.

4. In your terminal, press <kbd><kbd>Ctrl</kbd>+<kbd>C</kbd></kbd> to stop the `yarn start` command.

5. Run `git status`. On my system, `git status` will say this:

    ```console
    $ git status
    On branch main
    Your branch is up to date with 'upstream/main'.

    Changes not staged for commit:
      (use "git add <file>..." to update what will be committed)
      (use "git restore <file>..." to discard changes in working directory)
            modified:   yarn.lock

    Untracked files:
      (use "git add <file>..." to include in what will be committed)
            .yarn/
            .yarnrc.yml

    no changes added to commit (use "git add" and/or "git commit -a")
    ```

I’m using version 4.2.2 of Yarn if that makes a difference.

### PR DESCRIPTION
**Pull request by Jayman2000**
This is the specific revision of Node.gitignore that was used:
<https://github.com/github/gitignore/blob/8779ee73af62c669e7ca371aaab8399d87127693/Node.gitignore>

The previous .gitignore file was missing at least one folder that should have been ignored. Using a standard .gitignore file that multiple different people have contributed to will hopefully prevent us from forgetting things in the future.

Fixes #7.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,20 +1,132 @@
...
+.pnpm-debug.log*
 
-# Generated files
-.docusaurus
-.cache-loader
+# Diagnostic reports (https://nodejs.org/api/report.html)
+report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
 
-# Misc
-.DS_Store
```
Comment (ohadmamet) [inline .gitignore:12]: We should leave this in for people using MacOS machines.

File: .gitignore [hunk 2]
```diff
@@ -1,20 +1,132 @@
-# Dependencies
-/node_modules
+# This file was copied from <https://github.com/github/gitignore/blob/main/Node.gitignore>.
 
-# Production
-/build
```
Comment (ohadmamet) [inline .gitignore:5]: I am pretty sure this is a Docusaurus specific directory. We would not want static builds commited after local testing.

File: .gitignore [hunk 3]
```diff
@@ -1,20 +1,132 @@
...
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+lerna-debug.log*
+.pnpm-debug.log*
 
-# Generated files
-.docusaurus
-.cache-loader
```
Comment (ohadmamet) [inline .gitignore:9]: webpack generated?

**Review by ohadmamet — COMMENTED** [review]
Looks good but would like for some stuff to be checked out. See comments.

**Review by MassiveBreaker — APPROVED** [review]
lgtm

**Pull request merged**